scripts/jenkins/ardana/ansible/ardana-ses-manila.yml

```diff
@@ -0,0 +1,157 @@
+#
+# (c) Copyright 2018 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+- name: Configure ceph mds for manila
```

Comment:
Assuming we will have support for the ceph manila back-end in Crowbar too, can't you make this part of `cloud-ses-integration.yml` instead ? To make things simpler, you could even configure the SES cluster for manila regardless of whether it has been enabled or not, or whether Crowbar is deployed or Ardana.

Reply:
This means you will have to store the manila SES configuration (keyring ?) in something other than an ansible variable, which would correspond to what we want from the SES team, which is that the SES salt runner should configure manila too and return the result.

Reply:
I like that idea.

I have made the changes and now I am testing them...

Reply:
Changes tested in:
- cloud 8: https://ci.suse.de/blue/organizations/jenkins/openstack-ardana/detail/openstack-ardana/4260/pipeline/153

- cloud 9: https://ci.suse.de/blue/organizations/jenkins/openstack-ardana/detail/openstack-ardana/4259/pipeline